### DISCUSSION TITLE
Derived burrito discussion

### DISCUSSION THREAD

**Discussion by jtauber** [bible-technology/scripture-burrito#237] [Next]
To be implemented in #222 

Also see #209 #210 #214

**Comment by jtauber** [comment]
What would be helpful is a list of examples (and non-examples) of derived burrito use cases.

One example that seems to often be given is:

1. a subsetting of a scripture text (e.g. just the gospels)

but what are others?

2. translations?
3. "performances"? (e.g. audio)

What else?

What flavor-restrictions are there on derived burritos? Do they ONLY exist between burritos of the scripture type? Presumably not as presumably other flavor types could be subset or translations.

A NON-example is that derived burritos aren't about versioning. If I take a burrito and modify its ingredients, that does not make it a derived burrito per se. Any kind of change log or linking to previous versions is NOT what derived burritos are for.

PLEASE AGREE OR DISAGREE WITH THE ABOVE AND/OR ADD NEW CLARIFICATIONS IN THIS DISCUSSION

**Comment by jonathanrobie** [comment]
One of the major use cases for derived burritos is DBL, which licenses
translation projects for publication and provides content according to a
given license.  If a Paratext translation project is uploaded to DBL, what
is licensed is normally something derived from the project, omitting
project information not relevant to publishers.  In addition to omitting
that information, a license might include study notes or not, it might
include a set of images or not, etc., and the resulting burrito needs to
contain the license and content appropriate for that license.

**Reply by jtauber** [reply]
so I guess this just a different kind of subsetting. Not a subsetting the scope of scripture itself but subsetting the parascriptural elements included, for example, possibly with different licensing as you say.

**Comment by jonathanrobie** [comment]
It may not be subsetting, it could be some other kind of transformation.

**Reply by jtauber** [reply]
Presumably additions too? Or it could combine information previously in separate resources.

**Comment by jtauber** [comment]
To be clear, though: we're actually talking about burritos that represent resources/projects derived from other resources/projects, rather than burritos derived from other burritos. Right?

**Comment by jonathanrobie** [comment]
Yes.  In general, the translation project is a resource that has its own
version control system, and it becomes the basis for publication resources
in some publication system. That may include, for instance, a reading by a
voice actor, artwork added for a particular publication, a license designed
for a particular publication, etc.
